weaviate/collection/collection.py

```diff
@@ -24,7 +24,7 @@ def __init__(
 
         self.config = _ConfigCollection(self._connection, name)
         self.data = _DataCollection(connection, name, consistency_level, tenant)  # type: ignore
-        self.query = _GrpcCollection(connection, name, tenant)
+        self.query = _GrpcCollection(connection, name, tenant, consistency_level)
```

Comment:
This is a pedantry but can the order of `consistency_level` and `tenant` be uniform between `_DataCollection` and `_GrpcCollection`, unless it's a requirement?

Reply:
Sure, I know software-engineering-OCD ;)